ui/src/app/applications/components/application-summary/application-summary.tsx

```diff
@@ -529,6 +529,39 @@ export const ApplicationSummary = (props: ApplicationSummaryProps) => {
 
                             {app.spec.syncPolicy && app.spec.syncPolicy.automated && (
                                 <React.Fragment>
+                                    <div className='row white-box__details-row'>
+                                        <div className='columns small-12'>
+                                            <div className='checkbox-container'>
+                                                <Checkbox
+                                                    onChange={async (val: boolean) => {
+                                                        const confirmed = await ctx.popup.confirm(
+                                                            val ? 'Enable Auto-Sync?' : 'Disable Auto-Sync?',
+                                                            val
+                                                                ? 'If checked, application will automatically sync when changes are detected'
+                                                                : 'Are you sure you want to disable automated application synchronization'
+                                                        );
+                                                        if (confirmed) {
+                                                            const updatedApp = JSON.parse(JSON.stringify(props.app)) as models.Application;
+                                                            const currentAutomated = updatedApp.spec.syncPolicy?.automated || {prune: false, selfHeal: false};
+
+                                                            updatedApp.spec.syncPolicy = updatedApp.spec.syncPolicy || {};
+                                                            updatedApp.spec.syncPolicy.automated = {
+                                                                prune: currentAutomated.prune,
+                                                                selfHeal: currentAutomated.selfHeal,
+                                                                enabled: val
+                                                            };
+
+                                                            await updateApp(updatedApp, {validate: false});
+                                                        }
+                                                    }}
+                                                    checked={!!app.spec.syncPolicy?.automated?.enabled}
```

Comment:
Could you check this again?  The enabled state doesn't seem to be updated and it's always unchecked for me.